sample-app/src/main/res/values/constants.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
```

Comment:
`<!-- You'll set these fields during the Installation and Initialization steps of the documentation.-->`